### DISCUSSION TITLE
Install RITA on SO 2.4.50

### DISCUSSION THREAD

**Discussion by MPI-CSL-DH** [Security-Onion-Solutions/securityonion#12419] [2.4]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

0

### RAM

0

### Storage for /

0

### Storage for /nsm

0

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello,

Curios what is the recommended way to install RITA on SO 2.4.50?
There is a repo with an Installation-Script on git https://github.com/weslambert/securityonion-rita but that is from 2022. The standard Installation-Script from the official RITA repo failed, maybe cuz the OS of 2.4 changed. 

[-] Ensuring lsb-release is installed...
Failed to install package: redhat-lsb-core

[!] Installation FAILED on line 767.

So it seems the only Option would be the Docker-Installation?

Thx for your help.
Best Regards,
DH

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by InfosecGoon** [comment]
The cleanest approach would be to copy the Zeek logs to another server running RITA (use rsync in a crontab) and then push the results from RITA back into Security Onion for parsing, rather than installing it directly on the SO box.
